arlas-hits-exporter/export_data.bash

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
+set -o errexit -o pipefail
+
+usage(){
+	echo "Usage: ./export_data.bash --search_url==X --include=Y --format=Z"
+	echo " --search_url            Search request url which result is exported. Put the url between double or simple quotes."
+	echo " --include               Comma separated field names/paths to include in the exported file"
+	echo " --format                Export format : json | csv"
+	echo " --sort_csv              Columns of csv to sort on  : -k1 -k2"
+	echo " --output                Folder where exported data is generated"
+	exit 1
+}
+
+for i in "$@"
+do
+case "$i" in
+    --search_url=*)
+    SEARCH_URL="${i#*=}"
+    shift # past argument=value
+    ;;
+    --include=*)
+    INCLUDE="${i#*=}"
+    shift # past argument=value
+    ;;
+    --format=*)
+    FORMAT="${i#*=}"
+    shift # past argument=value
+    ;;
+    --sort_csv=*)
+    SORT_CSV="${i#*=}"
+    shift # past argument=value
+    ;;
+    --output=*)
+    OUTPUT="${i#*=}"
+    shift # past argument=value
+    ;;
+    *)
+    # unknown option
+    ;;
+esac
+done
+
+if [ -z ${SEARCH_URL} ]; then echo "--search_url argument is missing"; usage; else echo "### Your request is ==> ${SEARCH_URL}"; fi
+if [ -z ${INCLUDE} ]; then echo "--include argument is missing" & usage; else echo "### Fields to export ==> ${INCLUDE}"; fi
+if [ -z ${OUTPUT} ]; then OUTPUT=$PWD; echo "### Output file ==> ${OUTPUT}" else echo "### Output file ==> ${OUTPUT}"; fi
+
+## CONCATENATE FIELDS TO INCLUDE TO THE URL
+SEARCH_URL=${SEARCH_URL}"&include="${INCLUDE}
```

Comment:
Variables should be included in the double quotes, to safely manage white spaces:

```bash
"${SEARCH_URL}&include=${INCLUDE}"
```